Src/Fido2.Development/StoredCredential.cs

```diff
@@ -52,7 +52,7 @@ public class StoredCredential
 
     public byte[] UserId { get; set; }
 
-    public PublicKeyCredentialDescriptor Descriptor { get; set; }
```

Comment:
For less allocations we could cache the instance of the PublicKeyCredentialDescriptor as a private member.